core/src/mindustry/ui/dialogs/CustomGameDialog.java

```diff
@@ -24,8 +24,6 @@ public CustomGameDialog(){
     }
 
     void setup(){
-        clearChildren();
-        add(titleTable);
```

Comment:
doesn't sound perfectly safe, then again i don't know the gui code that well 🤔

Reply:
Indeed. It completely breaks layout.